src/utils/mavenUtils.ts

```diff
@@ -38,9 +38,17 @@ export async function rawEffectivePom(pomPath: string, options?: {cacheOnly?: bo
     return await readFileIfExists(outputPath);
 }
 
-export async function rawDependencyTree(pomPath: string): Promise<string | undefined> {
-    const outputPath: string = getTempFolder(pomPath);
-    await executeInBackground(`dependency:tree -Dverbose -DoutputFile="${outputPath}"`, pomPath);
+// export async function rawDependencyTree(pomPath: string): Promise<string | undefined> {
+//     const outputPath: string = getTempFolder(pomPath);
+//     await executeInBackground(`dependency:tree -Dverbose -DoutputFile="${outputPath}"`, pomPath);
+//     return await readFileIfExists(outputPath);
+// }
+export async function rawDependencyTree(pomPath: string): Promise<any> {
+    let outputPath: string;
+    {
+        outputPath = `${path.dirname(pomPath)}\\target\\dependency-graph.txt`;

```

Comment:
is `outputPath` used?

Reply:
yes, read file from `outputPath` and return. But there is no need to use `{}`

Reply:
Instead use `outputDirectory` and `outputFileName` in the command.